lib/server.js

```diff
@@ -224,9 +256,16 @@ module.exports = function createServer(config, log) {
               unblock: result.unblock,
               suspect: result.suspect
             })
-            res.send(result)
+            res.send({
+              block: result.block,
+              retryAfter: result.retryAfter,
+              unblock: result.unblock,
+              suspect: result.suspect
+            })
 
-            if (config.reputationService.enable && result.block) {
+            if (config.reputationService.enable
```

Comment:
This could be added to class as well, maybe as `reputationService.report(ip, result)`